scikeras/wrappers.py

```diff
@@ -703,6 +812,38 @@ class KerasClassifier(BaseWrapper):
         }
     )
 
+    _meta_params = (
+        BaseWrapper._meta_params.copy()
+    )  # parameters created by wrappers within `fit`
+    _meta_params.update(
```

Comment:
Thank you for the suggestion, this is cleaner, but we should do it the other way around (so that the child class can overwrite the base class):
```python3
_meta_params = {
     **BaseWrapper._meta_params
     "n_classes_", "target_type_", ...
}
```

Reply:
Does it matter? `_meta_params` is a set, so the size of the set can only grow. This might be a little cleaner:

``` python
_meta_params = BaseWrapper._meta_params.union({"n_classess_", "target_type_", ...})
```

Reply:
You are totally right, as you see above I was thinking of dicts... for sets it should not matter (for dicts it does)

Reply:
I see where my confusion came from: `_tags` is used in almost the exact same way but is a dict.

Reply:
Fixed and implemented as you suggested (I did end up using the order I suggested, but only so it matches `_tags` which requires that order).